app/wiring/AppComponents.scala

```diff
@@ -47,14 +48,16 @@ class AppComponents(context: Context, stage: String) extends BuiltInComponentsFr
 
   private val authAction = new AuthAction[AnyContent](authConfig, routes.Login.loginAction(), controllerComponents.parsers.default)(executionContext)
 
+  private val runtime = new DefaultRuntime {}
+
   override lazy val router: Router = new Routes(
     httpErrorHandler,
     new Application(authAction, controllerComponents),
     new Login(authConfig, wsClient, requiredGoogleGroups, googleGroupChecker, controllerComponents),
-    new SwitchesController(authAction, controllerComponents, stage),
-    new ContributionTypesController(authAction, controllerComponents, stage),
-    new AmountsController(authAction, controllerComponents, stage),
-    new EpicTestsController(authAction, controllerComponents, wsClient, stage),
+    new SwitchesController(authAction, controllerComponents, stage, runtime),
+    new ContributionTypesController(authAction, controllerComponents, stage, runtime),
+    new AmountsController(authAction, controllerComponents, stage, runtime),
+    new EpicTestsController(authAction, controllerComponents, wsClient, stage, runtime),
```

Comment:
Perhaps could avoid feeding this new dependency through by changing the implementation of the controller action, something like [this](https://github.com/loicdescotte/play-zio/blob/master/app/controllers/UserController.scala#L19) that uses [this implementation](https://github.com/loicdescotte/play-zio/blob/master/app/libs/http.scala#L19).

Reply:
Nice! I'll try this in a follow up PR